src/Sylius/Bundle/OrderBundle/spec/Form/Type/OrderTypeSpec.php

```diff
@@ -39,8 +39,19 @@ function it_is_a_form_type()
 
     function it_builds_form_with_items_collection_field(FormBuilderInterface $builder)
     {
-        $builder->add('items', 'collection', Argument::any())
-            ->willReturn($builder);
+        $builder
```

Comment:
IMO writing specs for form types is just additional work without any profit, but I'm not gonna be lobbying it right now :tada: